.gitignore

```diff
@@ -0,0 +1,13 @@
+# Ignore generated NPM files
+# These are sometimes included but not necessary for this project
+package-lock.json
```

Comment:
I would recommend keeping the lock file in version control

Reply:
Sorted